train_agent.py

```diff
@@ -1,20 +1,22 @@
 import click
 import os
-from PIL import Image
+from PIL import Image, ImageDraw
 import numpy as np
 from text_localization_environment import TextLocEnv
 import chainer
 import chainerrl
 import logging
 import sys
-
+from tb_chainer import SummaryWriter
+import time
 
 @click.command()
 @click.option("--steps", "-s", default=2000, help="Amount of steps to train the agent.")
 @click.option("--gpu/--no-gpu", default=False)
 @click.option("--imagefile", "-i", default='image_locations.txt', help="Path to the file containing the image locations.", type=click.Path(exists=True))
 @click.option("--boxfile", "-b", default='bounding_boxes.npy', help="Path to the bounding boxes.", type=click.Path(exists=True))
-def main(steps, gpu, imagefile, boxfile):
+@click.option("--tensorboard/--no-tensorboard", default=True)
```

Comment:
@bakoe Add instructions to README.md on how to display the results using TensorBoard